cfme/configure/configuration/__init__.py

```diff
@@ -602,8 +602,8 @@ class SMTPSettings(Updateable):
             ('port', Input("smtp_port")),
             ('domain', Input("smtp_domain")),
             ('start_tls', Input("smtp_enable_starttls_auto")),
-            ('ssl_verify', Select("select#smtp_openssl_verify_mode")),
-            ('auth', Select("select#smtp_authentication")),
+            ('ssl_verify', AngularSelect("smtp_openssl_verify_mode")),
```

Comment:
taking a look at the test results, im going to assume this one was perhaps working in 5.5 and early 5.6 and just never fixed